dist/vue-tables.js

```diff
@@ -3378,8 +3378,8 @@ module.exports = function (h) {
     perpageValues.push(h(
       "option",
       {
-        attrs: { value: value },
```

Comment:
I'm not 100% why the builder changed lines like this. Let me know if I should build this differently or at all before merging